src/main/java/edu/harvard/iq/dataverse/FieldType.java

```diff
@@ -4,5 +4,5 @@
  * The set of possible metatypes of the field. Used for validation and layout.
  */
 public enum FieldType {
-    TEXT, TEXTBOX, DATE, EMAIL, URL, FLOAT, INT, CHECKBOX, NONE
+    TEXT, TEXTBOX, DATE, EMAIL, URL, FLOAT, INT, CHECKBOX, NONE, LONG
```

Comment:
Introducing new FieldType requires more than just adding value in enum and handle it in solr (for example we need to handle validation of such field). I would not introduce for now new field type.